EMS/Helpers/DepartmentEnum.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace EmployeeManagement;
+public enum Department
```

Comment:
File name can be simply Department. Doesn't have to be departmentEnum. Fix it for all enum files.